src/App.js

```diff
@@ -0,0 +1,29 @@
+import { Route, Routes } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Instead of disabling the linter checks, I would strongly recommend you fix the linter error.

Reply:
hey @muneebulrehman! thanks a lot for the suggestion 👍 
actually, this disable has no effect, my project will work fine and will have no errors even without it 😀
I just forgot to remove it before submission. 😅